File: spider/src/main/java/com/javbus/spider/spider/entity/base/Actress.java
package com.javbus.spider.spider.entity.base;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;

/**
 * Actress
 */
@Data
public class Actress {
    private Integer id;
    private String name;
    @JsonProperty("actress_link")
    private String ActressLink;
    @JsonProperty("photo_link")
    private String photoLink;
    @JsonProperty("birth_day")
    private String birthDay;
    private String age;
    private String height;
    private String cup;
    private String bust;
    private String waist;
    private String hip;
    @JsonProperty("birth_place")
    private String birthPlace;
    private String hobby;
    @JsonProperty("is_censored")
    private Boolean isCensored;
}

File: javbus/utils/attrs/Actress.py
class Actress:
    name = ""
    actress_link = ""
    photo_link = ""
    birth_day = ""
    age = ""
    height = ""
    cup = ""
    bust = ""
    waist = ""
    hip = ""
    birth_place = ""
    hobby = ""
    is_censored = ""

    def __str__(self):
        return (
            f"Name: {self.name}\n"
            f"Actress Link: {self.actress_link}\n"
            f"Photo Link: {self.photo_link}\n"
            f"Birth Day: {self.birth_day}\n"
            f"Age: {self.age}\n"
            f"Height: {self.height}\n"
            f"Cup: {self.cup}\n"
            f"Bust: {self.bust}\n"
            f"Waist: {self.waist}\n"
            f"Hip: {self.hip}\n"
            f"Birth Place: {self.birth_place}\n"
            f"Hobby: {self.hobby}"
        )

    def toDict(self):
        return self.__dict__
